addOns/authhelper/src/main/java/org/zaproxy/addon/authhelper/AuthUtils.java

```diff
@@ -1153,7 +1148,7 @@ public static boolean isRelevantToAuth(HttpMessage msg) {
      * given URL.
      */
     public static void checkLoginLinkVerification(
-            HttpSender authSender, User user, WebSession session, String loginUrl) {
```

Comment:
Hopefully no external code is using this.

Reply:
Shall I restore?

Reply:
I'm fine if not, we can see if something breaks.